src/components/Section.js

```diff
@@ -10,19 +10,19 @@ export default class Section {
     // a CSS class selector where you'll add the card elements.
   }
 
-  renderItems() {
+  renderItems(items) {
     //   //It should iterate through the items array and call the renderer() function on each item.
     //   //This method should be called once on page load.
-    this._items.forEach((item) => {
+    items.forEach((item) => {
       const element = this._renderer(item);
       this._container.append(element);
     });
   }
 
   // takes a DOM element and adds it to the container.
   // This method should be called when adding an individual card to the DOM.
-  addItem({ name, link }) {
-    const element = this._renderer({ name, link });
+  addItem({ name, link, _id }) {
+    const element = this._renderer({ name, link, _id });
```

Comment:
**Можно лучше**


No need to destructure `{ name, link, _id }` because you make the same object in line 25

```js

  addItem(data) {
    const element = this._renderer(data);
```